app/routes.py

```diff
@@ -1,2 +1,55 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+# 1. Create a virtual environment and activate it
+# 1. Install the dependencies
+# 1. Define a `Planet` class with the attributes `id`, `name`, and `description`, and one additional attribute
+# 1. Create a list of `Planet` instances
+
+class Planet:
+    def __init__(self, id, name, description, color):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.color = color
+    
+    def to_json(self):
+        return {"id": self.id,
+                "name": self.name,
+                "description": self.description,
+                "color": self.color
+                }
+
+earth = Planet(1,"Earth","home planet","blue, brown, green, and white")
+mercury = Planet(2, "Mercury", "closest planet to the sun", "grey")
+neptune = Planet(3, "Neptune", "eight planet, farthest from sun", "blue")
+jupiter = Planet(4, "Jupiter", "fifth planet from the sun, largest planet", "brown, orange and tan")
+
+planets = [earth, mercury, neptune, jupiter]
+
+planets_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+@planets_bp.route("", methods = ["GET"])
+def get_all_planets():
```

Comment:
Nice job naming the method to describe what this route does.

In the future, if you end up splitting your routes up into distinct route files (cat_routes.py and dog_routes.py for example in a routes directory) then you could have method names like get_all() and get_one() since we know that all the routes in cat_routes.py are related to the Cat class.